README.md

```diff
@@ -86,6 +86,11 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Entrust](https://entrust.wd1.myworkdayjobs.com/EntrustCareers/job/United-States---Sunrise-FL/Software-Engineering-Intern_R002369) | Sunrise, FL | Software Engineer Intern |
 | [Enphase Energy](https://jobs.jobvite.com/enphase-energy/job/oFPRmfwr) | Austin, TX | Embedded Firmware Engineer Intern |
 | [Körber](https://jobs.koerber.com/pharma/job/Fargo-Software-Developer-Intern-ND-58102/954269201/) | Fargo, ND | Software Developer Intern |
+| [BenefitMall](https://truist.wd1.myworkdayjobs.com/CRC/job/Dallas-TX/BenefitMall---Software-Development-Intern--Hybrid-_R0077897) | Hybrid (Dallas, TX) | Software Development Intern |
```

Comment:
This one (BenefitMall) is interesting because it states
> Assignment length is multiple semesters

implying that this may not just be a summer internship. I would move this to the off-season internship page, but it's up to you.

Reply:
Yeah, I thought the posting was pretty vague. I agree on moving it to off-season.